content/help/ops/security/repairing-citadel/index.md

```diff
@@ -22,3 +31,17 @@ debugging information and check if there are any errors:
 $ kubectl logs -l istio=citadel -n istio-system
 $ kubectl describe pod -l istio=citadel -n istio-system
 {{< /text >}}
+
+If you want to check a workload (with `default` service account and `default` namespace)
+certificate's lifetime, run:
+
+{{< text bash >}}
+$ kubectl get secret -o json istio.default -n default | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -noout -text | grep "Not After" -C 1
+  Not Before: Jun  1 18:23:30 2019 GMT
+  Not After : Aug 30 18:23:30 2019 GMT
+Subject:
+{{< /text >}}
+
+Remember to replace `istio.default` and `-n default` with `istio.YourServiceAccount` and
```

Comment:
this should be an info block.

Reply:
Not sure whether you mean {{< text bash >}}. I don't find info level at: https://github.com/istio/istio.io/blob/533a3efc0e5d6a66f03f28552785ac24fe3a9996/content/test/callouts/index.md

Reply:
I meant `{{< info >}}`.  As you point out, that isn't in the callouts :).  Can you use `{{< tip >}}`?

Cheers
-steve

Reply:
Done